### ISSUE TITLE
Add Bluetooth function

### ISSUE THREAD

**Issue by louies0623** [helloSystem/hello#191]
Add Bluetooth function e.g.
- Bluetooth connection 
- Bluetooth file transfer 
- Bluetooth control For example: mouse with Bluetooth function, keyboard, etc. can control the computer 
- Bluetooth audio or microphone

**Comment by probonopd** [comment]
A __Bluetooth__ applet similar to the one made by @linuxkettle for the Battery would need to be written. Volunteers?

**Comment by linuxkettle** [comment]
Well I'm CCed in now, I'll give it a go.

**Comment by probonopd** [comment]
For getting started with Bluetooth file transfer, the following works for me:
https://forums.ghostbsd.org/viewtopic.php?t=970

I tested this successfully up to the point where the instructions say "If Bluetooth is working you'll get your phone discovered and see output like this".

I had to turn Bluetooth off and on again on my phone to make it discoverable.

However, I don't want users to deal with the command line and configuration files.

Hence I tried to build the Qt Bluetooth File Transfer Example application that comes as an example with Qt Creator. It would be pretty perfect for our purposes, but unfortunately it says `qt.bluetooth: Dummy backend running. Qt Bluetooth module is non-functional.`

How can we get Qt to interact with FreeBSD's Bluetooth stack?
